backend/lambda_functions/registartion_form/registration_form_lambda.py

```diff
@@ -0,0 +1,83 @@
+import json
+import psycopg2
+import os
+import boto3
+
+# Environment Variables
+REGION = os.environ['REGION']
+DB_HOST = os.environ['DB_HOST']
+DB_NAME = os.environ['DB_NAME']
+DB_USER = os.environ['DB_USER']
+DB_PASSWORD = os.environ['DB_PASSWORD']
+DB_PORT = os.environ['DB_PORT']
+WHATSAPP_LINK = "https://chat.whatsapp.com/Bx3hIysSqmG5p3ZYtnNXw7"
+
+# Initialize the SES to send the email
+ses = boto3.client('ses', region_name=REGION)
+
+def lambda_handler(event, context):
+    # extract the data from the event
+    user_data = json.loads(event['body'])
+    name = user_data['name']
+    email = user_data['email']
+    phone = user_data['phone']
+
+    # Connect to the database
+    try:
+        conn = psycopg2.connect(
+            host=DB_HOST,
+            database=DB_NAME,
+            user=DB_USER,
+            password=DB_PASSWORD,
+            port=DB_PORT
+        )
+        cursor = conn.cursor()
+
+        # Inserting userdata into db (assumes table is already created and called community_members)
```

Comment:
think about creating a general execution method for later function development (maybe more queries?)